### PR TITLE
Add codestyle linters

### PR DESCRIPTION
**Pull request by okucheruk**
This PR adds linters for avoiding bikeshedding about code style.

Highlights:
- JavaScript static code analysis with ESLint.
- HTML static code analysis with HTMLHint.
- Bootstrap static code analysis with Bootlint.
- CSS static code analysis with Stylelint.
- YAML static code analysis with Yamllint.

### CODE REVIEW

File: app/view/element/Input.js [hunk 1]
```diff
@@ -47,19 +52,27 @@ export default class Input
...
     withValue(value) {
 
         this.getElement().value = value;
         return this;
     }
 
+    /**
+     *
+     * @return {String}
+     */
```
Comment (dkalinin) [inline app/view/element/Input.js]: Please don't leave JSDoc empty.

File: .eslintrc.yml [hunk 2]
```diff
@@ -17,4 +19,9 @@ parserOptions:
   ecmaVersion: 2018
   sourceType: module
 
-rules: {}
+rules:
+  indent: ["error", 4, {SwitchCase: 1}]
+  max-len: ["error", 100]
+  padded-blocks: [
+    "error", {classes: 'always'}
+  ]
```
Comment (dkalinin) [inline .eslintrc.yml]: The purpose of having ESLint is to have a common code style convention. Please use google config only (which should be added as a dependency) with only max-len overridden.

File: .circleci/config.yml [hunk 3]
```diff
@@ -22,9 +22,15 @@ jobs:
           name: "Install dependencies"
           command: |
             sudo apt-get update
-            sudo apt-get -y install python3-pkg-resources yamllint shellcheck
+            sudo apt-get -y install python3-pkg-resources \
+                                    yamllint \
+                                    shellcheck
             sudo npm install -g npm@latest --no-package-lock
-            sudo npm install -g eslint
+            sudo npm install -g eslint \
```
Comment (dkalinin) [inline .circleci/config.yml]: There's no need to install ESLint globally.

**Comment by dkalinin** [comment]
@okucheruk How about adding ESLint run to CI?

File: app/view/element/Input.js (hunk 1 above)
Comment (okucheruk) [reply app/view/element/Input.js]: JSDocs added.

File: .eslintrc.yml (hunk 2 above)
Comment (okucheruk) [reply .eslintrc.yml]: Padded-blocks setting removed.

**Comment by okucheruk** [comment]
@dkalinin All linters already integrated into CI.
Makefile provides equal automations for local manual jobs and CI jobs as well.

File: .eslintrc.yml (hunk 2 above)
Comment (dkalinin) [reply .eslintrc.yml]: Ok, here's what I meant:
* Remove all from rules except max-len.
* Remove 'eslint:recommended' from `extends`.
* Remove .eslint-config-google.js file.
* Add eslint-config-google to devDependencies.

Comment (okucheruk) [reply .eslintrc.yml]: @dkalinin 
Jfyi.
There are several rules in the `eslint:recommended` ruleset that Google style is not opinionated about that I might want to enforce in my project.

To use Google style in conjunction with ESLint's recommended rule set, extend them both, making sure to list google last. (c)google

Google uses 'eslint:recommended' for own projects also. E.g.`'extends': ['eslint:recommended', 'google', 'plugin:shaka-rules/config'],`

Comment (dkalinin) [reply .eslintrc.yml]: Sure, and some Google projects don't use that at all: 
https://github.com/GoogleChrome/puppeteer/blob/master/.eslintrc.js

Comment (okucheruk) [reply .eslintrc.yml]: Fixed.

File: .circleci/config.yml (hunk 3 above)
Comment (okucheruk) [reply .circleci/config.yml]: fixed

**Comment by okucheruk** [comment]
@dkalinin, fixed, PTAL.

File: app/view/element/Element.js [hunk 4]
```diff
@@ -18,127 +18,121 @@
...
+  }
+
+  /**
+   * Renders element or web component.
+   *
+   * @protected
+   * @abstract
+   */
+
+  /* eslint-disable */
```
Comment (dkalinin) [inline app/view/element/Element.js]: Why is it disabled?

Comment (okucheruk) [reply app/view/element/Element.js]: This code disabled due to abstration.
Now it's refactored to more concrete rule:
`_render(parentElement) { // eslint-disable-line no-unused-vars`

**Pull request merged**